recipes-kernel/linux/linux-qoriq-sdk.bbappend

```diff
@@ -2,6 +2,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 SRC_URI_append += "file://${MACHINE}/dts                     \
                    file://${MACHINE}/kconfig                 \
                    file://patches/0001-Backport-PPC64-patch.patch \
+                   file://patches/0002-Chage-to-fit-T600-NOR-flash-partition.patch \
```

Comment:
This should probably be moved down to the commented out SRC_URI_append_t600 since only applies to T600.